index.html

```diff
@@ -37,7 +44,33 @@
         "breadcrumb": []
       }, {
         "@id": "#libraries",
-        "breadcrumb": []
+        "breadcrumb": [{
+          "@id": "/data-model/",
+          "name": "@rdfjs/data-model",
+          "description": "Basic implementation of RDFJS Data Model.",
+          "implements": [ "DataFactory" ]
+        }, {
+          "@id": "/dataset/",
+          "name": "@rdfjs/dataset",
+          "description": "Basic implementation of RDFJS Dataset.",
+          "implements": [ "Dataset" ]
+        }, {
+          "@id": "/N3.js/",
+          "name": "n3",
+          "description": "Lightning fast, spec-compatible, streaming RDF for JavaScript.",
+          "implements": [ "DataFactory", "Store", "Sink" ]
```

Comment:
what does it mean to implement Sink?

Reply:
I guess that would be needed to indicate parsers.

Reply:
https://github.com/rdfjs/N3.js#interface-specifications

So Actually no Store interface, I'll remove it.